package.json

```diff
@@ -33,6 +33,7 @@
     "prop-types": "^15.7.2",
     "react": "^16.8.2",
     "react-dom": "^16.10.2",
+    "react-toast-notifications": "^2.4.3",
```

Comment:
@irmerk do we have a component for toasts? Or are we using something else for toast in any other project?

Regardless, I think we should use [this package](https://www.npmjs.com/package/react-semantic-toasts) as we are already using semantic-ui so makes more sense to be consistent with styling.

Reply:
@algomaster99 
Should I go with the package you mentioned?

Reply:
Yes, go ahead!